private/dev/package.json

```
@@ -7,7 +7,8 @@
     }
   },
   "dependencies": {
-    "@uppy/companion": "workspace:^"
+    "@uppy/companion": "workspace:^",
+    "@uppy/locales": "workspace:^"
```

I don't think `@uppy/locales` should be special, don't we want to load it from source like the other plugins?
```suggestion
    "@uppy/companion": "workspace:^"
```

---

The other plugins come from the bundle and locale does not exist there. We didn't want to add locales to the bundle either because it would bloat the bundle size even further while you wouldn't use more than one locale most likely.

---

I don't think that's correct, we use Vite aliases to resolve directly to the source, it should not go through the bundle at all 🤔
https://github.com/transloadit/uppy/blob/c7bb57b7f8a690fa79a711fd3f5346b963be71a0/private/dev/vite.config.js#L28-L41

---

I tried to apply this suggestion locally, and couldn't see any thing failing. I also tried removing the bundle, and didn't notice anything broken 🤷‍♂️